rcl/test/rcl/test_timer.cpp

```diff
@@ -39,21 +36,10 @@ class TestTimerFixture : public ::testing::Test
     rcl_node_options_t node_options = rcl_node_get_default_options();
     ret = rcl_node_init(this->node_ptr, name, "", &node_options);
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string_safe();
-    set_on_unexpected_malloc_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED MALLOC";});
-    set_on_unexpected_realloc_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED REALLOC";});
-    set_on_unexpected_free_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED FREE";});
-    start_memory_checking();
```

Comment:
Was the motivation for removing the use of memory tools from some tests because that info's captured elsewhere? Or something else?

Reply:
Most of the time the check was meaningless or memory tools wasn't being used at all (it was setup, but never asserted that memory functions should not be called).